model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -765,6 +765,9 @@ message ProcessBundleSplitRequest {
 
     // TODO(SDF): Allow providing weights rather than sizes.
     // TODO(SDF): Allow specifying allowed/preferred split points.
+
+    // A set of allowed indices where the SDK may split.
+    repeated int64 allowed_split_points = 3;
```

Comment:
Put right after fraction_of_remainder. Also note that if this is empty, there is no constraint.

Reply:
done